packages/next/build/webpack/plugins/nextjs-require-cache-hot-reloader.ts

```diff
@@ -1,15 +1,20 @@
-import {Compiler} from 'webpack'
+import { Compiler, Plugin } from 'webpack'
+import { realpathSync } from 'fs'
 
-function deleteCache (path: string): void {
-  delete require.cache[path]
+function deleteCache (path: string) {
+  try {
+    delete require.cache[realpathSync(path)]
+  } catch(e) {
+    // probably doesn't exist
```

Comment:
Should we make sure it doesn't exist here? Any other errors should probably be shown as it'd indicate breakage.